### DISCUSSION TITLE
如何实现类似这样的链接桩

### DISCUSSION THREAD

**Discussion by Weilesszz** [antvis/X6#3417] [Q&A] [answered]
![自定义节点](https://user-images.githubusercontent.com/83270566/227079095-2d4076cc-5e37-4b8a-93c8-6399774e4e74.png)
文字需要自定义，并且适配它的宽度。使用的是vue3

**Comment by news-ttt** [comment, marked as answer]
使用自定义节点样式, 可以参考这个地址： https://x6.antv.antgroup.com/tutorial/intermediate/vue

然后连接桩你可以配置connecting中allowNode为true，这样你就不用定义连接桩就可以对你的节点的边进行连接了

**Reply by Weilesszz** [reply]
感谢回复，最开始也有思考过用嵌套父子节点来代替这个链接桩，但是后来因为想要方便导出所以没有使用。
看来比较好的方法也许只能是这样了
